### PR TITLE
v19 release notes

### PR DESCRIPTION
**Pull request by adnasa**
Release notes for #2041

### CODE REVIEW

File: website/src/releases/2021-04/release-v19.mdx [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+date: 2021-04-08
+title: Custom Applications v19
+description: |
+  The Application Kit packages have been released as a new major version v19.\
```
Comment (tdeekens) [inline website/src/releases/2021-04/release-v19.mdx:5]:
```suggestion
  The Application Kit packages have been released with a new major version v19.\
```

File: website/src/releases/2021-04/release-v19.mdx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  In this release note we go through some migration steps.
+type: announcement
+topics:
+  - Breaking Changes
+  - Dependencies
+---
+
+The Application Kit packages have been released as a new **major version** `v19`.
```
Comment (tdeekens) [inline website/src/releases/2021-04/release-v19.mdx:13]:
```suggestion
The Application Kit packages have been released with a new **major version** `v19`.
```

**Review by emmenko — APPROVED** [review]

File: website/src/releases/2021-04/release-v19.mdx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+In this version there are no explicit changes in the packages but mainly updates to major versions of some of the dependency libraries. For instance:
```
Comment (tdeekens) [inline website/src/releases/2021-04/release-v19.mdx:15]:
```suggestion
This version does not contain explicit changes in our packages but mainly updates to major versions of some of the depended upon libraries. For instance:
```

File: website/src/releases/2021-04/release-v19.mdx [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+## Constraints
+
+For the `<Constraints.Horizontal />` component, the deprecated `constraint` prop has been removed. Now only the `max` prop is used instead.
+
+```diff
+- <Constraints.Horizontal constraint="m" />
++ <Constraints.Horizontal max={7} />
+```
+
+The equivalent mapping from `constraint` to `max` is as following:
```
Comment (tdeekens) [inline website/src/releases/2021-04/release-v19.mdx:37]: 💯 This is awesome!

File: website/src/releases/2021-04/release-v19.mdx [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+Furthermore, some peer dependencies have been updated to only require more recent versions. Please refer to each specific package's `CHANGELOG.md`.
+
+<!--more-->
+
+# Migrating UI Kit packages
+
+## Constraints
+
+For the `<Constraints.Horizontal />` component, the deprecated `constraint` prop has been removed. Now only the `max` prop is used instead.
```
Comment (tdeekens) [inline website/src/releases/2021-04/release-v19.mdx:30]:
```suggestion
For the `<Constraints.Horizontal />` component, the deprecated `constraint` prop has been removed. Now only the `max` prop can be used instead.
```

File: website/src/releases/2021-04/release-v19.mdx [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+## Text
+
+For the `<Text.Headline>` and `<Text.Subheadline>` components, the deprecated `elementType` prop has been removed. Now only the `as` prop is used instead.
+
+```diff
+- <Text.Headline elementType="h2" intlMessage={message} />
++ <Text.Headline as="h2" intlMessage={message} />
+```
+
+For the `<Text.Body>` and `<Text.Detail>` components, the deprecated `isInline` prop has been removed. Now only the `as` prop is used instead.
```
Comment (tdeekens) [inline website/src/releases/2021-04/release-v19.mdx:56]:
```suggestion
For the `<Text.Body>` and `<Text.Detail>` components, the deprecated `isInline` prop has been removed. Now only the `as` prop can be used instead.
```

File: website/src/releases/2021-04/release-v19.mdx [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+
+## Link
+
+For the `<Link>` component, the deprecated `hasUnderline` prop has been removed. Links now always have an **underline** style applied.
+
+When the prop `isExternal` is used and the `to` prop is not a **string** but a **location object**, an error is thrown.
+
+## Tag
+
+For the `<Tag>` component, the deprecated `linkTo` prop has been removed. Now only the `to` prop is used instead.
```
Comment (tdeekens) [inline website/src/releases/2021-04/release-v19.mdx:71]:
```suggestion
For the `<Tag>` component, the deprecated `linkTo` prop has been removed. Now only the `to` prop can be used instead.
```

File: website/src/releases/2021-04/release-v19.mdx [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+
+## SecondaryButton
+
+For the `<SecondaryButton>` component, the deprecated `linkTo` prop has been removed. Now only the `to` prop is used instead.
```
Comment (tdeekens) [inline website/src/releases/2021-04/release-v19.mdx:75]:
```suggestion
For the `<SecondaryButton>` component, the deprecated `linkTo` prop has been removed. Now only the `to` prop can be used instead.
```

**Review by tdeekens — COMMENTED** [review]
Thanks for writing it up. Great change log.

Left some ideas. All nits I think as suggestions so they're easy to take of leave.

**Review by tdeekens — APPROVED** [review]

**Pull request merged**